client/app/states/services/details/details.html

```diff
@@ -116,6 +130,21 @@
                     <input class="form-control" disabled value="${{ ::vm.service.chargeback.used_cost_sum | number:3 }}"/>
                   </div>
                 </div>
+                <div class="form-group">
```

Comment:
EEEEEK 🕷 🕸 
Oh wait, its inline style, still just as terrifying, this should be refactored to live in the sass, ping me if you need an 👂